src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -236,7 +236,28 @@ object ModelJsonProtocol extends WorkspaceJsonSupport {
   implicit val impLibrarySearchResponse = jsonFormat4(LibrarySearchResponse)
   implicit val impLibraryBulkIndexResponse = jsonFormat3(LibraryBulkIndexResponse)
 
-  implicit val impDuosConsent = jsonFormat10(Consent)
+  implicit object impDuosDataUse extends RootJsonFormat[DuosDataUse] {
+    override def write(ddu: DuosDataUse): JsValue = {
+      val existingProps: Seq[(String, JsValue)] = ddu.getClass.getDeclaredFields map { f =>
+        f.setAccessible(true)
+        f.get(ddu) match {
+          case Some(x: Boolean) => f.getName -> x.toJson
+          case Some(y: String) => f.getName -> y.toJson
+          case Some(ls: Seq[_]) =>
+            ls.head match {
+              case (a: String) => f.getName -> (a +: ls.tail.map(_.asInstanceOf[String])).toJson
+              case _ => f.getName -> JsNull
+            }
```

Comment:
I think you can one-line the `Seq` case, with even more type-checking that avoids `asInstanceOf`:
```
case Some((h:String) :: t) => f.getName -> (h +: t.collect{case z:String => z}).toJson
```